packages/components/select/src/select.stories.ts

```diff
@@ -49,39 +51,40 @@ export default {
           ${slot?.() ??
           html`
             <sl-select
+              ?clearable=${clearable}
```

Comment:
Clicking on the `label` in the `form-field` should focus the select component (select-button), right?
Just like it works with other form fields.
Right now it's not working.